src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php

```diff
@@ -187,6 +215,30 @@ public function deleteAction($id)
     }
 
     /**
+     * Delete an filter with the given id.
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function cdeleteAction(Request $request)
+    {
+        $ids = explode(',', $request->get('ids'));
+        if ($ids && count($ids) > 0) {
+            try {
+                $this->getManager()->batchDelete($ids);
```

Comment:
how is it handled if one of the requested does not exists? will the rest be deleted with 404 / 200 or will nothing be deleted with 404?